doc/devel/development_setup.rst

```diff
@@ -164,12 +164,23 @@ command ::
 
 The 'editable/develop mode' builds everything and places links in your Python environment
 so that Python will be able to import Matplotlib from your development source directory.
-This allows you to import your modified version of Matplotlib without re-installing after
-every change. Note that before the merging of the `Meson port
-<https://github.com/matplotlib/matplotlib/pull/26621>`_, this is only true for ``*.py``
-files. If you change the C-extension source based on a commit before the change to the
-Meson build system (which might also happen if you change branches), you will have to
-re-run the above command.
+This allows you to import your modified version of Matplotlib without having to
+re-install after changing a ``.py`` or compiled extension file. On Windows, you
+may need to :external+meson-python:ref:`vsenv-example`.
```

Comment:
Why is this generating warnings?

Reply:
I'm not sure if that's a usual thing right now. On my current-version Git-for-Windows, Git Bash is in MSYS2, but there are no compilers installed.

Reply:
The move to troubleshooting seems good.